packages/react-composites/tests/app/chat/LiveTestApp.tsx

```diff
@@ -58,23 +58,23 @@ export const LiveTestApp = (): JSX.Element => {
   React.useEffect(() => {
     if (adapter && uploadedFiles.length) {
```

Comment:
Should this be `uploadedAttachments`?

Reply:
actually i was wrong, this is nothing to do with File[], i just updated this to `uploadAttachments`